src/langchain_google_alloydb_pg/alloydb_vectorstore.py

```diff
@@ -729,6 +730,12 @@ def max_marginal_relevance_search_with_score_by_vector(
         )
         return self.engine._run_as_sync(coro)
 
+    async def set_maintenance_work_mem(self, num_leaves: int, vector_size: int) -> None:
+        index_memory_required = 50 * num_leaves * vector_size * 4
+        await self.engine._aexecute(
+            f"SET maintenance_work_mem = {index_memory_required};"
```

Comment:
We need to confirm this is the right format. I see examples using 'TO' instead of '=' and also setting the measurement unit like 'GB'

Reply:
updated with units